Cargo.toml

```diff
@@ -159,19 +159,20 @@ url = "2.5.4"
 [profile.release]
 codegen-units = 1
 lto = true
+debug = false
```

Comment:
nit: I believe this is superfluous - https://doc.rust-lang.org/cargo/reference/profiles.html#debug

Reply:
Agree